daylight.gemspec

```diff
@@ -24,11 +24,12 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec'
   s.add_development_dependency 'rspec-rails',    '~> 2.14.0'
   s.add_development_dependency 'simplecov-rcov', '~> 0.2.3'
-  s.add_development_dependency 'webmock',        '~> 1.18.0'
+  s.add_development_dependency 'webmock',        '~> 1.21.0'
   s.add_development_dependency 'sqlite3'
   s.add_development_dependency 'factory_girl'
   s.add_development_dependency 'faker'
   s.add_development_dependency 'rake'
+  s.add_development_dependency 'pry'
```

Comment:
Not sure if this should be a dependency on Daylight itself. Might be better if you just `gem install pry` on the system-level.